reconcile/openshift_base.py

```diff
@@ -257,7 +258,9 @@ def realize_data(dry_run, oc_map, ri,
                  take_over=False,
                  caller=None,
                  wait_for_namespace=False,
-                 no_dry_run_skip_compare=False):
+                 no_dry_run_skip_compare=False,
+                 override_enable_deletion=None,
```

Comment:
why this is not `True/False`?

Reply:
whatever the value the user passes is the override value. if a user doesn't pass it, they don't want to override. see the check `if not None` below which matches this idea.